src/pages/Profile/Profile.css

```diff
@@ -13,6 +13,60 @@
          This will help you when you get started.
     */
 
+.text {
+    /* flex-direction: column; */
+    margin: auto;
+    width: 800px;
+}
+
+#circle img {
+    /* display: flex;
+    justify-content: left; */
+    height: 125px;
+    margin: auto;
+    
+
+}
+
+.name {
+    font-size: 27px;
```

Comment:
it's good to have a `line-height` attribute alongside any `font-size` that isn't the pre-defined one so that your text will have room to space itself out; figma will usually have this in the text style properties too